src/puptoo/process/profile.py

```diff
@@ -69,6 +69,7 @@ def catch_error(parser, error):
         InstalledProductIDs,
         Specs.branch_info,
         Specs.tags,
+        Specs.pmlog_summary,
```

Comment:
Not sure how this works but instead of using `Specs.pmlog_summary` can we use parser `from insights.parsers.pmlog_summary import PmLogSummary` here and push the parsed content with metadata?